package.json

```diff
@@ -119,7 +121,14 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run test:changed"
+      "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "src/**/*.{tsx,ts}": [
+      "npm run test:staged",
+      "prettier --write",
+      "eslint --fix"
+    ]
```

Comment:
Do we also need to run `git add` if eslint or prettier do some changes to files?

Reply:
maybe here has been some update but it complained when I tried to add it. It said that it does the adding for me

Reply:
Ok, maybe it was some old example which I saw

Reply:
Yeah I have used git add in the past